metrics/pcp/pcp.go

```diff
@@ -0,0 +1,129 @@
+package pcp
+
+import (
+	"github.com/go-kit/kit/metrics"
+	"github.com/performancecopilot/speed"
+)
+
+// Reporter encapsulates a speed client
```

Comment:
Thanks for the method comments! Please use complete sentences, including proper punctuation and grammar, i.e.

```go
// Reporter encapsulates a speed client.
```